modules/org.kie.kogito.rhpam.bundleinstall/module.yaml

```diff
@@ -1,5 +1,5 @@
 name: org.kie.kogito.rhpam.bundleinstall
-version: "7.13.0"
```

Comment:
I'm not sure about this one, should it be updated?

Reply:
yes it should IMO